netaudio/console/commands/subscription/_add.py

```diff
@@ -98,7 +98,7 @@ async def subscription_add(self):
         if rx_device and not tx_device:
             tx_device = rx_device
 
-        if rx_channel and rx_device and tx_channel and tx_channel:
+        if rx_channel and rx_device and tx_channel and tx_device:
```

Comment:
```suggestion
        if rx_channel and rx_device and tx_channel:
```

The variable `tx_device` is checked to be falsy earlier on line 98, then populated if so on line 99. Thus, this check looks to be redundant.

In the case that `rx_device` is not set, then this conditional statement will resolve to `False` before it gets as far as checking `tx_device`.